src/decays/specializations/Ah/decay_Ah_to_ubaru.inc

```diff
@@ -143,5 +143,21 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, bar<UpTypeQuark>::type, U
       result = (1-4.*xOS)*result_DR + 4*xOS*result_OS;
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
+       static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_lilith)) ||
+       static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::calculate_normalized_effc))
+   ) {
+      neutral_higgs_effc.add_coupling(field_as_string<PseudoscalarHiggs>(indexIn),
+         {
+            boost::hana::unpack(bar<UpTypeQuark>::pdgids, _to_array<bar<UpTypeQuark>::numberOfGenerations>).at(indexOut1.at(0)),
+            boost::hana::unpack(UpTypeQuark::pdgids, _to_array<UpTypeQuark::numberOfGenerations>).at(indexOut2.at(0))
+         },
+         std::pair<std::string, std::complex<double>> {
+            field_as_string<PseudoscalarHiggs>(indexIn) + "-" + field_as_string<bar<UpTypeQuark>::type>(indexOut1) + "-" + field_as_string<UpTypeQuark>(indexOut2),
+            4.*std::max(xDR, xOS) > 1. ? std::imag(AhQQbarVertexDR_P)*1i : 1i*std::sqrt(result)
```

Comment:
Could you please add a short comment here to explain this expression?

Reply:
Done in 0659a0388bc16c58a1e9fb0110016ede73be7233.